ravenframework/JobHandler.py

```diff
@@ -977,6 +977,8 @@ def terminateJobs(self, ids):
       @ In, ids, list(str), job prefixes to terminate
       @ Out, None
     """
+    #XXX terminateJobs modifies the running queue, which
+    # cleanJobQueue, and fillJobQueue assume can't happen
```

Comment:
does this comment need to be rephrased due to the fixes in this PR?

Reply:
Possibly it needs to be changed to a warning.